chains/miraclechain/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://explorer-miracle-chain-dqfxgxhidy.t.conduit.xyz/api/v2
+    family: blockscout
+    name: Miracle Chain explorer
+    url: https://explorer.miracleplay.io
+chainId: 92278
```

Comment:
also need a `blocks` config above this e.g. https://github.com/hyperlane-xyz/hyperlane-registry/blob/a296c25fb9f308cc9d7ba5c91331d982e0f1ec81/chains/arbitrum/metadata.yaml#L7-L10